app/assets/stylesheets/application.scss

```diff
@@ -33,6 +33,21 @@ footer {
   padding-top: 15px;
 }
 
+.page-nav {
+  margin-top: 35px;
+  padding-left: 15px;
+  background-color: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.